apps/api/src/app/inversify.config.ts

```diff
@@ -12,6 +12,9 @@ import {
   usdRepositorySymbol,
 } from '@cowprotocol/repositories';
 
+const DEFAULT_CACHE_VALUE_SECONDS = ms('2min') / 1000; // 2min cache time by default for values
+const DEFAULT_CACHE_NULL_SECONDS = ms('30min') / 1000; // 30min cache time by default for NULL values (when the repository don't know)
```

Comment:
```suggestion
const DEFAULT_CACHE_NULL_SECONDS = ms('30min') / 1000; // 30min cache time by default for NULL values (when the repository isn't known)
```

Reply:
I will do in this PR to avoid merge conflicts https://github.com/cowprotocol/bff/pull/75

Reply:
0282d7b3af53e43c1bcedea7f0fcc07821fa71f5